packages/react-portal/src/components/Portal/renderPortal.tsx

```diff
@@ -0,0 +1,13 @@
+import * as ReactDOM from 'react-dom';
+import { PortalState } from './Portal.types';
+
+/**
+ * Render the final JSX of Portal
+ */
+export const renderPortal = (state: PortalState) => {
+  if (state.mountNode) {
```

Comment:
```suggestion
  if (state.mountNode && !isSSR) {
```

Should we add SSR check there?

Reply:
The only guard here is `React.createPortal` if a user does not guard their own logic to create a DOM node it should blow up long before it gets here.

If a user does guard their own logic to create a DOM node, their input should be `null/undefined` since the prop only accepts `HTMLElement` as type. 

Is there another case i have not thought of ?

Reply:
> If a user does guard their own logic to create a DOM node, their input should be `null/undefined` since the prop only accepts `HTMLElement` as type.
> 
> Is there another case i have not thought of ?

Relaxing types to accept `null`/`undefined` sounds reasonable for this case 👍

Reply:
This is already an optional prop so `undefined` can be accepted, will also relax for `null`